src/events/types.ts

```diff
@@ -74,6 +74,7 @@ export type ApprovedItemEventBusPayload = BaseEventBusPayload &
       | 'topic'
       | 'isSyndicated'
       | 'createdBy'
+      | 'authors'
```

Comment:
Nit pick: Don't have to do it right now but I was just thinking for future cases like this, we can use the `Omit<T,K>` utility type and omit the ones we don't need vs picking all the ones we need

Reply:
very fair point, and we do use `Omit` elsewhere. i was just following the path of least resistance here.